nilearn/interfaces/fmriprep/load_confounds.py

```diff
@@ -206,10 +206,19 @@ def load_confounds(
         DVARS only.
 
     fd_threshold : :obj:`float`, default=0.2
+
+        .. deprecated:: 0.11.0
+           The default value will be changed to 0.5 in 0.x.
```

Comment:
```suggestion
           The default value will be changed to 0.5 in 0.13.0
```
if this gets merged before our next release